Lectures/Lecture1_theOpenScienceEthos.md

```diff
@@ -39,35 +40,32 @@ In this lecture we will discuss the advancement of the open science movement in
 ## Evaluation and tutorials
 + During the introductory lecture, students are asked to collect the terms and concepts they have heard about for the first time, in this lecture. During the breakout discussions, students are asked to reflect on why they have not learned about these concepts before and the barriers they see in realization of open science ideals.
```

Comment:
We don't have a breakout discussion as such, do we? Instead, students move to their workgroup rooms. Make that more explicit here?

Reply:
let us discuss this. I am on the fence if this should be for the workgroup rooms or in the lecture hall. I understand that it will be chaotic in the lecture hall. Workgroups could be a better option if we find a way to share back the outcome/summary of each group discussion with the rest of the participants